src/panels/config/integrations/ha-integration-card.ts

```diff
@@ -453,6 +453,9 @@ export class HaIntegrationCard extends LitElement {
           cursor: pointer;
           min-height: 35px;
         }
+        ha-svg-icon {
```

Comment:
You only want to target the more info icon right? Now you also target the overflow icon.
```suggestion
        mwc-list-item ha-svg-icon {
```

Reply:
I wanted both, to ensure the theme, but the overflow is this color already so it does not really matter